crates/sui-indexer/src/lib.rs

```diff
@@ -82,6 +91,11 @@ impl IndexerConfig {
             url.port().unwrap_or_default()
         )
     }
+
+    /// returns all endpoints for which we have implemented on the indexer
+    pub fn all_migrated_methods() -> Vec<String> {
+        self.migrated_methods = migrated_methods.iter().map(|&s| s.to_string()).collect();
```

Comment:
self.migrated_methods

Reply:
also can you add a comment here that this is used for local / CI testing only, meanwhile production `migrated_methods` should parse from CLI?